.github/workflows/build.yml

```diff
@@ -732,6 +731,55 @@ jobs:
           SLACK_MESSAGE: Failure with initialising QA deployment for ${{env.APPLICATION}}
           SLACK_WEBHOOK: ${{ steps.keyvault-yaml-secret.outputs.SLACK-WEBHOOK }}
 
+  test_aks:
```

Comment:
Should we add continue-on-error so it doens't fail risk failing the main pipeline?

Reply:
done